pkg/controller/controller.go

```diff
@@ -146,24 +148,28 @@ func NewController(dnsSuffix string, istioClient *crd.Client, k8sClient kubernet
 	processor := processor.NewController(configStoreCache)
 	crcController := onboarding.NewController(configStoreCache, dnsSuffix, serviceIndexInformer, crcResyncInterval, processor)
 	adIndexInformer := adInformer.NewAthenzDomainInformer(adClient, 0, cache.Indexers{})
-	apController := authzpolicy.NewController(configStoreCache, serviceIndexInformer, adIndexInformer, istioClientSet, apResyncInterval, enableOriginJwtSubject, apDryRun)
+	apController := &authzpolicy.Controller{}
```

Comment:
Should we just set this to nil instead by default to minimize memory? Using `var apController *authzpolicy.Controller`